getting-started/exploring-labs.md

```diff
@@ -0,0 +1,86 @@
+# Setup [](setup)
+Now that you’ve been invited to Timescale Labs, you’re ready to work 
+with some data. The first thing to do is to create a new database. 
+Timescale Labs is a PostgreSQL database with TimescaleDB extensions 
+already installed. Timescale Labs automatically provisions and manages 
+a TimescaleDB instance in Amazon Web Services.
+
+>:WARNING: Timescale Labs is in private alpha. [Request an invite]: https://labs.timescale.com/. We will keep this document updated as we add features, but the screenshots and instructions may be temporarily out of date.
+
+## Sign in to Timescale Labs [](signin)
+Sign in to Timescale Labs and you’ll see that a `timescale_demo` 
+service is automatically created for you. Click on the service 
+and you’ll see a screen similar to this:
+
+<img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/labs_config_screen.png" alt="timescale labs config screen"/>
+
+## Connect to your Timescale Labs database [](connect)
+To connect to the database, you’ll need to make sure the `psql` 
+utility is installed on your command line. Follow the instructions for 
+your platform in order to 
+[setup the psql command-line utility](https://blog.timescale.com/tutorials/how-to-install-psql-on-mac-ubuntu-debian-windows/).
+
+Take note of the hostname and password (blacked out in the screenshot 
+above) for your Timescale Labs instance.
+
+Connecting to your Timescale database is as easy copying and pasting 
+your service URL. Note that the link you copied from the start screen 
+will look different than the one below; example-password, example-host 
```

Comment:
variables like 'example-password' should be set as inline code with single backticks:
```suggestion
will look different than the one below; `example-password`, `example-host` 
```